### PR TITLE
Fixes for qc-workflow.sh

### PR DESCRIPTION
**Pull request by davidrohr**
@chiarazampolli @knopers8 : QC tasks should not be added solely by the presense of the detector, but only if requested in the list of QC tasks. Note that the default list of QC tasks is the list available detectors + $LIST_OF_GLORECO.

### CODE REVIEW

File: DATA/production/qc-workflow.sh [hunk 1]
```diff
@@ -87,8 +87,8 @@ elif [[ -z $QC_JSON_FROM_OUTSIDE ]]; then
     [[ -z "$QC_JSON_CPV" ]] && QC_JSON_CPV=$O2DPG_ROOT/DATA/production/qc-sync/cpv.json
     [[ -z "$QC_JSON_PHS" ]] && QC_JSON_PHS=$O2DPG_ROOT/DATA/production/qc-sync/phs.json
     [[ -z "$QC_JSON_TRD" ]] && QC_JSON_TRD=$O2DPG_ROOT/DATA/production/qc-sync/trd.json
-    [[ -z "$QC_JSON_GLO_VTX" ]] && QC_JSON_GLO=$O2DPG_ROOT/DATA/production/qc-sync/glo-vtx-qcmn-epn.json
-    [[ -z "$QC_JSON_GLO_ITSTPC_MTCH" ]] && QC_JSON_GLO=$O2DPG_ROOT/DATA/production/qc-sync/glo-itstpc-mtch-qcmn-epn.json
+    [[ -z "$QC_JSON_GLO_PRIMVTX" ]] && QC_JSON_GLO=$O2DPG_ROOT/DATA/production/qc-sync/glo-vtx-qcmn-epn.json
```
Comment (chiarazampolli) [inline DATA/production/qc-workflow.sh:90]: Here there seems to be something wrong... we are redefining `QC_JSON_GLO` with the line below. Am I wrong?

File: DATA/production/qc-workflow.sh [hunk 2]
```diff
@@ -87,8 +87,8 @@ elif [[ -z $QC_JSON_FROM_OUTSIDE ]]; then
...
+    [[ -z "$QC_JSON_GLO_ITSTPC_MTCH" ]] && QC_JSON_GLO=$O2DPG_ROOT/DATA/production/qc-sync/glo-itstpc-mtch-qcmn-epn
```
Comment (chiarazampolli) [inline DATA/production/qc-workflow.sh:91]: Why did you remove the `.json`?

Comment (davidrohr) [reply DATA/production/qc-workflow.sh:91]: aehm, no idea how that happened... will fix it

File: DATA/production/qc-workflow.sh (hunk 1 above)
Comment (davidrohr) [reply DATA/production/qc-workflow.sh:90]: indeed, that is wrong. But was wrong before, so it was never active ... :(

Comment (chiarazampolli) [reply DATA/production/qc-workflow.sh]: Yes, sure, it was wrong also before, I pointed it out since you were fixing the script (thanks!).
